docs/source/dev_guide/algorithms/logistic_regression/logistic-regression.rst

```diff
@@ -261,27 +263,29 @@ Examples
 
 .. tabs::
 
-  .. tab:: DPC++
-
-    Batch Processing:
-
-    - :ref:`log_reg_binary_dense_batch.cpp`
-    - :ref:`log_reg_dense_batch.cpp`
-
-  .. tab:: C++
+  .. tab:: C++ (CPU)
 
     Batch Processing:
 
     -  :cpp_example:`log_reg_dense_batch.cpp <logistic_regression/log_reg_dense_batch.cpp>`
     -  :cpp_example:`log_reg_binary_dense_batch.cpp <logistic_regression/log_reg_binary_dense_batch.cpp>`
 
   .. tab:: Java*
+  
+    .. note:: There is no support for Java on GPU.
 
     Batch Processing:
 
     -  :java_example:`LogRegDenseBatch.java <logistic_regression/LogRegDenseBatch.java>`
     -  :java_example:`LogRegBinaryDenseBatch.java <logistic_regression/LogRegBinaryDenseBatch.java>`
 
+  .. tab:: Python* with DPC++ support
+
+    Batch Processing:
+
+    - :daal4py_sycl_example:`log_reg_dense_batch.py`
+    - :daal4py_sycl_example:`log_reg_binary_dense_batch.py`
+
   .. tab:: Python*
 
```

Comment:
For python example missed log_reg_binary_dense_batch.py. See 294 line in this file

Reply:
fixed